app/src/main/java/com/example/hris/repository/HrisRepository.kt

```diff
@@ -11,15 +13,86 @@ import javax.inject.Singleton
 @Singleton
 class HrisRepository @Inject constructor(
     private val hrisDao: HrisDao
-){
+) {
 
-    val profileData: LiveData<User> = hrisDao.getProfile()
+    val loginData = MutableLiveData<User>()
+    val loginMessage = MutableLiveData<String>()
 
+    val userData = hrisDao.getProfile()
 
-    suspend fun refreshRepository(user: User) {
-        withContext(Dispatchers.IO) {
-            hrisDao.deleteAll()
-            hrisDao.insertProfile(user)
+    val timeLogs: LiveData<List<TimeLogs>> = hrisDao.getTimeLogs()
+
+    suspend fun insertProfile(username: String, password: String) {
```

Comment:
Apply the same logic as `addTimeLogs()`. Fix method name. Make sure describes what it actually does.